blockchain/stake/staketx.go

```diff
@@ -1313,9 +1313,11 @@ func FindSpentTicketsInBlock(block *wire.MsgBlock) *SpentTicketsInBlock {
 	revocations := make([]chainhash.Hash, 0, block.Header.Revocations)
 
 	for _, stx := range block.STransactions {
-		// This can only be called with treasury disabled.
-		const noTreasury = false
-		if IsSSGen(stx, noTreasury) {
+		// It is safe to use the version as a proxy for treasury activation here
+		// since version 3 votes were invalid prior to its activation and
+		// therefore could not have made it into a block.
```

Comment:
> version 3 votes were invalid prior to its activation

Is this accurate?  I thought it was only the script version that was enforced, and from a quick look, I can't find anything that was preventing version 3 votes prior to the treasury agenda activating.  Since the agenda already activated, as long as there were no version 3 transactions in the blockchain before that point, then this is fine anyway, but I was just wondering if this comment is accurate.

Reply:
That looks good to me.  Thanks for clarifying!

Reply:
Updated (both places).